test/llvmpasses/late-lower-gc-bugs.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: opt -enable-new-pm=0 -load libjulia-codegen%shlibext -LateLowerGCFrame -S %s
+; RUN: opt -enable-new-pm=1 --load-pass-plugin=libjulia-codegen%shlibext -passes='function(LateLowerGCFrame)' -S %s
+
+; Old bugs (should be caught by assertions and death)
+
+; This one tests that PHI nodes don't have duplicated bitcasts for the same BB
+define {} addrspace(10)* @"julia__generic_matmatmul!_927"() {
+top:
+  %0 = call {}*** @julia.get_pgcstack()
+  ret {} addrspace(10)* null
```

Comment:
The test case could use a bit of cleanup to at least link the CFG back together, otherwise they tend to be a bit fragile.